docs/modules/pipelines/pages/configuring-jobs.adoc

```diff
@@ -169,3 +174,39 @@ cluster change depending on these two settings.
 |suspend job
 |===
 
+== Configuring a User Code Namespace
+
+Normal Jet jobs can be configured to use a xref:clusters:user-code-namespaces.adoc[User Code Namespace] (UCN), which provides them
+access to custom resources associated with that UCN. {ucn} must be enabled and a namespace configured with the desired resources
+created before it can be used in jobs. Any data structures used in these jobs should also have their UCN configured to ensure
+operations for that data structure also have access to the necessary custom resources. Light jobs do not support UCN.
+
+{ucn} are managed separately to the Jet system, so these resources persist between individual jobs. This support allows for jobs to
+be un-deployed and re-deployed while still being able to use the same custom resources within pipelines.
```

Comment:
we do not use "un-deployed" and "re-deployed" in context of jobs, it is rather submit/update

Reply:
Correct, thanks: https://github.com/hazelcast/hz-docs/pull/1383/commits/70431358252c50d178a69ee6bd062a08a0ea0a8e